packages/xo-web/src/common/modal.js

```diff
@@ -289,7 +289,7 @@ export const confirm = ({ body, icon = 'alarm', title, strongConfirm }) =>
 let formModalState
 export const form = ({
   component,
-  defaultValue,
+  defaultValue = {},
```

Comment:
Why?

What does this fix?

Reply:
I've used `modal form` without `defaultValue`
```jsx
({ effects, install, multi, poolPredicate, value }) => (
    <Container>
        <SelectPool
          className='mb-1'
          multi={multi}
          onChange={effects.onChangePool}
          predicate={poolPredicate}
          required
          value={value.pools}
        />
  </Container>
```
As you see, in this case we will get `cannot get "pools" of undefined`
And I was stuck, because I taught that `value` should be initialized by default.

Reply:
If you don't use a default value for the form modal is logical that the value will be `undefined`.

What makes you don't use a default value for the component?

Reply:
Nothing prevents me to use `default value`. But in many cases, we don't need to set `defaultValue` and use `modal form`

Reply:
The `defaultValue` property is used to set the modal default value, if it's not set the value will be `undefined`.

And, we can't set a default value to a default value.

Reply:
Okey so I'll close the pull request :) thank you for your explanation.